src/integrators/inc_hyperelastic_integrator.cpp

```diff
@@ -6,119 +6,123 @@
 
 #include "inc_hyperelastic_integrator.hpp"
 
-double IncrementalHyperelasticIntegrator::GetElementEnergy(const mfem::FiniteElement &  el,
-                                                           mfem::ElementTransformation &Ttr, const mfem::Vector &elfun)
+namespace serac {
+
+double IncrementalHyperelasticIntegrator::GetElementEnergy(const mfem::FiniteElement&   el,
+                                                           mfem::ElementTransformation& Ttr, const mfem::Vector& elfun)
 {
   int    dof = el.GetDof(), dim = el.GetDim();
   double energy;
 
-  DSh.SetSize(dof, dim);
-  Jrt.SetSize(dim);
-  Jpr.SetSize(dim);
-  Jpt.SetSize(dim);
-  PMatI.UseExternalData(elfun.GetData(), dof, dim);
+  DSh_.SetSize(dof, dim);
+  Jrt_.SetSize(dim);
+  Jpr_.SetSize(dim);
+  Jpt_.SetSize(dim);
+  PMatI_.UseExternalData(elfun.GetData(), dof, dim);
 
-  const mfem::IntegrationRule *ir = IntRule;
+  const mfem::IntegrationRule* ir = IntRule;
   if (!ir) {
     ir = &(mfem::IntRules.Get(el.GetGeomType(), 2 * el.GetOrder() + 3));  // <---
   }
 
   energy = 0.0;
-  model->SetTransformation(Ttr);
+  model_->SetTransformation(Ttr);
   for (int i = 0; i < ir->GetNPoints(); i++) {
-    const mfem::IntegrationPoint &ip = ir->IntPoint(i);
+    const mfem::IntegrationPoint& ip = ir->IntPoint(i);
     Ttr.SetIntPoint(&ip);
-    CalcInverse(Ttr.Jacobian(), Jrt);
+    CalcInverse(Ttr.Jacobian(), Jrt_);
 
-    el.CalcDShape(ip, DSh);
-    MultAtB(PMatI, DSh, Jpr);
-    Mult(Jpr, Jrt, Jpt);
+    el.CalcDShape(ip, DSh_);
+    MultAtB(PMatI_, DSh_, Jpr_);
+    Mult(Jpr_, Jrt_, Jpt_);
 
     for (int d = 0; d < dim; d++) {
-      Jpt(d, d) += 1.0;
+      Jpt_(d, d) += 1.0;
     }
 
-    energy += ip.weight * Ttr.Weight() * model->EvalW(Jpt);
+    energy += ip.weight * Ttr.Weight() * model_->EvalW(Jpt_);
   }
 
   return energy;
 }
 
-void IncrementalHyperelasticIntegrator::AssembleElementVector(const mfem::FiniteElement &  el,
-                                                              mfem::ElementTransformation &Ttr,
-                                                              const mfem::Vector &elfun, mfem::Vector &elvect)
+void IncrementalHyperelasticIntegrator::AssembleElementVector(const mfem::FiniteElement&   el,
```

Comment:
Idea why there are extra spaces here that clangformat didnt remove? minor nitpick

Reply:
Probably to line up with the argument below it.